Userland/Libraries/LibJS/Tests/builtins/Temporal/Duration/Duration.compare.js

```diff
@@ -0,0 +1,112 @@
+describe("correct behavior", () => {
+    test("length is 2", () => {
+        expect(Temporal.Duration.compare).toHaveLength(2);
+    });
+
+    function compareDurations(duration1, duration2, expected) {
+        var result;
+        expect(() => {
+            result = Temporal.Duration.compare(duration1, duration2);
+        }).not.toThrowWithMessage(
+            RangeError,
+            "A starting point is required for balancing calendar units"
+        );
```

Comment:
This doesn't need to be wrapped in expect, it does nothing. if it throws, it fails anyway.